### ISSUE TITLE
BIOS check is case sensitive on case-insensitive filesystem (exfat/fat32)

### ISSUE THREAD

**Issue by nfriedly** [knulli-cfw/distribution#33]
### Knulli build version

40-dev-d8b86623f3 2024/05/30 01:50

### Your architecture

RG35XX Plus

### Issue description

I got a missing bios warning for `psxonpsp660.bin` despite having the correct file with the correct hash - turns out mine was named `PSXONPSP660.bin`. Changing the case fixed the warning, but I was surprised that was an issue on a case-insensitive filesystem.

### Detailed reproduction steps

Installed knuli using dd (ubuntu). Let it do it's first boot, then shut down. Formatted a second microsd on my laptop, moved it to my gr35xx+, booted, and let knuli prepare it. Rebooted knuli, then shut it down, removed the microsd, and moved it back to my laptop to copy over a few roms and bios files. Back to the 35xx+, booted up and launched a psx game. Got the missing bios warning. After a bit of trial-and-error, I figured out that renaming it to lower case solved the issue.

### Details of any attempts to fix this yourself

Renamed file to lower case

### Details of any modifications you have made to Knulli.

I deleted a few of the bundled games.

**Comment by nfriedly** [comment]
As an aside, I saw that https://knulli.org/faqs/#q-i-want-to-add-my-own-roms-to-the-sdcard-but-my-windowsmac-computer-cant-see-the-roms-partition but I confirmed that my second microsd is fat32. I know I formatted it that way on my laptop, just expecting that's what batocera would want. I don't know if knuly reformatted it as fat32 or just didn't bother reformatting it since it was already good enough.

I suppose pushing for case-sensitivity isn't such a bad thing if you're moving toward a case-sensitive filesystem. (Although, in that case I'd recommend btrfs ;)
